src/flask_pyoidc/provider_configuration.py

```diff
@@ -164,22 +181,45 @@ def ensure_provider_metadata(self):
     def registered_client_metadata(self):
         return self._client_metadata
 
-    def register_client(self, redirect_uris, extra_parameters=None):
+    def register_client(self, client: Client, redirect_uri: str):
+
```

Comment:
Extra empty line.

Reply:
Empty line after the function declaration is reserved for docstring.